src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -0,0 +1,359 @@
+package org.isf.medicalinventory.gui;
+
+import org.isf.utils.jobjects.InventoryState;
+import org.isf.generaldata.MessageBundle;
+import org.isf.utils.jobjects.ModalJFrame;
+
+import com.toedter.calendar.JDateChooser;
```

Comment:
Why are you not using our version  https://github.com/informatici/openhospital-gui/blob/develop/src/main/java/org/isf/utils/jobjects/GoodDateChooser.java ?